### PR TITLE
Updates to the AWS automated tests

### PR DESCRIPTION
**Pull request by leftwo**
Some updates so MacOS can run the scripts.
Added a missing export for ami_id variable.
Updated the README with a little more info.

### CODE REVIEW

File: aws_benchmark/driver.sh [hunk 1]
```diff
@@ -24,6 +24,8 @@ program_required terraform
 program_required virtualenv
 program_required python3
 program_required pip
+program_required aws
+program_required ansible-playbook
```
Comment (jmpesp) [inline aws_benchmark/driver.sh:28]: this should be installed by the `pip install "ansible==5.0.1"` line in `bring_up_resources.sh`, not required to run driver.sh

**Review by jmpesp — CHANGES_REQUESTED** [review]

File: aws_benchmark/driver.sh (hunk 1 above)
Comment (jmpesp) [reply aws_benchmark/driver.sh:28]: specifically I mean ansible-playbook, not aws :)

Comment (leftwo) [reply aws_benchmark/driver.sh]: done in cf0b91a

**Review by jmpesp — APPROVED** [review]

**Comment by leftwo** [comment]
@gjcolombo I'm going to push these updates, and we can iterate on it if you find more to add.  I want to build some more off it as well.

**Pull request merged**